mssql-jdbc-scenario keeps failing in GitHub Action

Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [x] Requirement
- [ ] Feature or performance improvement

### Requirement or improvement
Investigate and fix the test case.

Refer to 
- https://github.com/apache/skywalking/pull/7190/checks?check_run_id=2942325933#step:5:19687
- https://github.com/apache/skywalking/pull/7171/checks?check_run_id=2944222324#step:5:18920
```
  + mv ./target/mssql-jdbc-scenario.jar ./target/mssql-jdbc-scenario.jar.original ./target/mssql-jdbc-scenario.zip /home/runner/work/skywalking/skywalking/test/plugin/workspace/mssql-jdbc-scenario/6.2.2.jre8
  + java -jar -Xmx256m -Xms256m -Dconfigure.file=/home/runner/work/skywalking/skywalking/test/plugin/scenarios/mssql-jdbc-scenario/configuration.yml -Dscenario.home=/home/runner/work/skywalking/skywalking/test/plugin/workspace/mssql-jdbc-scenario/6.2.2.jre8 -Dscenario.name=mssql-jdbc-scenario -Dscenario.version=6.2.2.jre8 -Doutput.dir=/home/runner/work/skywalking/skywalking/test/plugin/workspace/mssql-jdbc-scenario/6.2.2.jre8 -Dagent.dir=/home/runner/work/skywalking/skywalking/test/plugin/../../skywalking-agent -Djacoco.home=/home/runner/work/skywalking/skywalking/test/plugin/../jacoco -Ddebug.mode= -Ddocker.image.version=jdk8-1.0.0 /home/runner/work/skywalking/skywalking/test/plugin/dist/plugin-runner-helper.jar
  + [[ 0 -ne 0 ]]
  + echo 'start container of testcase.name=mssql-jdbc-scenario-6.2.2.jre8'
  start container of testcase.name=mssql-jdbc-scenario-6.2.2.jre8
  + bash /home/runner/work/skywalking/skywalking/test/plugin/workspace/mssql-jdbc-scenario/6.2.2.jre8/scenario.sh
  Creating network "mssql-jdbc-scenario-jdk8-1.0.0" with the default driver
  Pulling mssql-server (microsoft/mssql-server-linux:)...
  manifest for microsoft/mssql-server-linux:latest not found: manifest unknown: manifest unknown
  docker startup failure!
  Error: Process completed with exit code 1.
```

--- a/test/plugin/scenarios/mssql-jdbc-scenario/configuration.yml
+++ b/test/plugin/scenarios/mssql-jdbc-scenario/configuration.yml
@@ -23,7 +23,7 @@ depends_on:
   - mssql-server
 dependencies:
   mssql-server:
-    image: microsoft/mssql-server-linux
+    image: mcr.microsoft.com/mssql/server
     hostname: mssql-server
     expose:
       - "1433"

--- a/test/plugin/scenarios/mssql-jtds-scenario/configuration.yml
+++ b/test/plugin/scenarios/mssql-jtds-scenario/configuration.yml
@@ -23,7 +23,7 @@ depends_on:
   - mssql-server
 dependencies:
   mssql-server:
-    image: microsoft/mssql-server-linux
+    image: mcr.microsoft.com/mssql/server
     hostname: mssql-server
     expose:
       - "1433"

